inst/app/server.R

```diff
@@ -814,6 +814,30 @@ ORDER BY gri.sample_id;
 
   # Samples Check-in --------------------------------------------------
 
+  output$check_in_preview <- renderTable({
+    # This table shows a preview of the loaded file
+    if(is.null(input$check_in_samples_file$datapath)){
+      loaded_file <- data.frame()
+    } else {
+      loaded_file <- readxl::read_excel(input$check_in_samples_file$datapath,
+                                        col_names = c("sample_id", "received_sample", "event", "entered_by", "verified_by", "alias", "comments"),
+                                        skip = 1)
+    }
+
+    return(loaded_file)
+  })
+
+  output$check_in_file_submission <- renderTable({
+    # This table shows if new samples are created in check in process
+    if(is.null(input$check_in_samples_file$datapath)){
+      samples_created <- data.frame()
+    } else {
+      samples_created <- grunID::check_in_jpe_field_samples(con, input$check_in_samples_file$datapath)
```

Comment:
don't submit this data twice, we already do this in the `observeEvent` below this code, notice that I store the samples created in a reactive `samples_created_from_checkin`
```r
samples_created_from_checkin(samples_created)
```
so instead use this reactive to determine if new samples were created, if they were then show this table. Potentially use the `validate` function again to either show table or message.